components/post_view/post_image/post_image.jsx

```diff
@@ -103,9 +110,15 @@ export default class PostImageEmbed extends React.PureComponent {
     };
 
     render() {
+        const imageDimentions = getFileDimentionsForDisplay(this.props.dimentions, {maxHeight: 500, maxWidth: 450});
```

Comment:
I know I'm getting annoying but the same here with the second arg, make it a constant

Reply:
As i was saying in the other comment

benefits of defining this as a constant(if you meant constants file) outweighs the cons in longer run. If we start to see a pattern we should probably resolve to a constant then.

I think We should stick to the scope for a constants like this i.e probably just define them in render function